Corona-Warn-App/src/deviceForTesters/res/navigation/test_nav_graph.xml

```diff
@@ -169,5 +169,19 @@
         android:id="@+id/vaccinationTestFragment"
         android:name="de.rki.coronawarnapp.vaccination.ui.VaccinationTestFragment"
         android:label="VaccinationTestFragment"
-        tools:layout="@layout/fragment_test_vaccination" />
+        tools:layout="@layout/fragment_test_vaccination" >
+        <action
+            android:id="@+id/action_vaccinationTestFragment_to_vaccinationDetailsFragment"
+            app:destination="@id/vaccinationDetailsFragment" />
+    </fragment>
+
+    <fragment
+        android:id="@+id/vaccinationDetailsFragment"
+        android:name="de.rki.coronawarnapp.vaccination.ui.details.VaccinationDetailsFragment"
+        android:label="fragment_vaccination_details"
+        tools:layout="@layout/fragment_vaccination_details">
+        <argument
+            android:name="certificateId"
```

Comment:
`vaccinationCertificateId` ? Because we also have `ProofCertificate` which will probably also have an ID.